PowerShellEditorServices.build.ps1

```diff
@@ -313,30 +313,55 @@ task RestorePsesModules -After Build {
     # Save each module in the modules.json file
     foreach ($moduleName in $moduleInfos.Keys)
     {
-        if (Test-Path -Path (Join-Path -Path $submodulePath -ChildPath $moduleName))
+        $moduleInstallPath = (Join-Path -Path $subModulePath -ChildPath $moduleName)
+
+        if (-not (Test-Path -Path $moduleInstallPath))
```

Comment:
I noticed that you changed the logic from:

```
if ( the path existed ) {
   continue
}

blah
blah
blah
```
to
```
if ( the path did not exist ) {
   blah
   blah
   blah
} else {
   it existed
}
```

The first organization is a bit cleaner and doesn't require an extra indent for a solid amount of code. Can you refactor it back?